addons/website/static/src/snippets/s_image_gallery/options.js

```diff
@@ -157,23 +158,29 @@ options.registry.GalleryLayout = options.registry.CarouselHandler.extend({
      */
     _slideshow() {
         const imageEls = this._getItemsGallery();
+        const imgHolderEls = this._getImgHolderEls();
         const images = Array.from(imageEls).map((img) => ({
             // Use getAttribute to get the attribute value otherwise .src
             // returns the absolute url.
             src: img.getAttribute('src'),
-            alt: img.getAttribute('alt'),
         }));
         var currentInterval = this.$target.find('.carousel:first').attr('data-bs-interval');
         var params = {
             images: images,
```

Comment:
We could replace this by `images: imagesEls`. However, if we do that, the small carousel thumbnails would have the absolute url of the images in their `style` attribute.

Reply:
You could give `imageEls` and then in the qweb rendering do `imgEl.getAttribute('src')`, let's do that :)